extensions/attention-stream/src/youtube/utils/traverseFishing.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Recrusive tree traversal of JSON object:
+ * Returns or deletes one or multiple keys
+ * depending on parameters
+ */
+const traverse = (
```

Comment:
Instead of exporting functions as const I suggest we change it to:
`export function traverse....` since it can then appear in stack traces. Same for other functions below.